client/components/sub-masterbar-nav/README.md

```diff
@@ -0,0 +1,37 @@
+Sub Masterbar Nav
+=================
+
+This component displays a navigation header under the masterbar.
+
+### How to use:
+
+The component should be inserted into a ```Main``` component.  
+Options should contain a ```label``` and an ```uri```. If the latter matches the ```uri``` property of the component, it will be marked as selected.  
+In case no option is selected, the ```fallback``` value will be shown in the dropdown.
+
+```icon``` should be a valid gridicon name. The icon will be displayed next to/on top of the label.  
+If no ```icon``` is provided, ```star``` will be used by default.
+
+```js
+const SubMasterbarNav = require( 'components/sub-masterbar-nav' );
+
+const options = {
```

Comment:
Nitpick -- should be an array (not an object) to be consistent with the implementation, and https://github.com/Automattic/wp-calypso/pull/11903/files#diff-6e3d613e7ebf414e5f0aadde79da1794R32.